### PR TITLE
PROP-10: proptoken-webapp and propetoken-enterprise-webapp should use same API server

### PR DESCRIPTION
**Pull request by sakshigokhru**
### Description
_proptoken-webapp and proptoken-enterprise-webapp should use same API server_

### CODE REVIEW

File: config/custom-environment-variables.json [hunk 1]
```diff
@@ -15,10 +15,15 @@
...
-    "integratorKey": "DOCUSIGN_INTEGRATOR_KEY"
+    "integratorKey": "DOCUSIGN_INTEGRATOR_KEY",
+    "webhookEventUrl": "DOCUSIGN_WEBHOOK_EVENT_URL"
   },
   "USPS": {
     "userId": "USPS_USER_ID",
     "server": "USPS_SERVER"
-  }
+  },
+  "sendgrid": {
```
Comment (bbbneo333) [inline config/custom-environment-variables.json]: You could've defined `sendgrid.emailFrom` and `sendgrid.emailFromName` in `default.json`? As these values are not changing across deployments?

File: src/routes/property.js [hunk 2]
```diff
@@ -2,10 +2,19 @@ const express = require('express');
...
-router.get('/', controller.getPropertyAddress);
-router.get('/download_signed_deed/:envelopeId', controller.donloadSignedDeed);
+router.get('/search', controller.searchPropertyAddress);
+router.get('/download_signed_deed/:envelopeId', controller.downloadSignedDeed);
 router.post('/verify_address', controller.verifyAddress);
+router.post('/send_for_sign', controller.sendDeedForSign);
+router.get('/', controller.getPropertyAddress);
+router.get('/details', controller.getPropertyDetails);
+router.put('/update/:id', controller.updateProperty);
+router.post('/deed_status', bodyParser.text({
```
Comment (bbbneo333) [inline src/routes/property.js]: Any middleware should not be a part of routing. You should simply inject `controller.deedStatus` here. And at the controller:
```js
exports.deedStatus = [bodyParser.text({limit: '50mb',
  type: '*/xml',
}), (req, res, next) {
  // controller logic here
}];

File: src/helpers/utils.js [hunk 3]
```diff
@@ -15,3 +18,99 @@ exports.parseMongooseError = (err) => {
 };
 
 exports.checkMongooseConnectionErr = err => err.message === 'no connection available for operation and number of stored operation > 0';
+
+exports.voidDocuSignContract = (res, envelopeId, cb) => {
```
Comment (bbbneo333) [inline src/helpers/utils.js]: Is the `res` object really needed here? Avoid passing stuff like `req` and `res` in utils.

Comment (bbbneo333) [reply src/helpers/utils.js]: Pass the `res.locals.DOCU_SIGN_ACCOUNT_ID` via parameter.

**Comment by sakshigokhru** [comment]
@bbbneo333 - I improved all the changes mentioned in the comments, please have a look.

**Pull request merged**